pvr.dvblink/changelog.txt

```diff
@@ -1,3 +1,6 @@
+[B]Version 4.5.0[/B]
+Updated to PVR API v5.8.0
```

Comment:
Shouldn't you mention that TVMosaic support was added?

Reply:
Added and merged. Thanks!